Makefile

```diff
@@ -55,20 +57,23 @@ INC_DIRS += $(UT_CORE_DIR)/src
 
 SRC_DIRS += $(UT_CORE_DIR)/src
 
-XLDFLAGS += -L $(UT_CONTROL)/lib -lut_control -Wl,-rpath-link,$(LIBWEBSOCKET_LIB_DIR)
+XLDFLAGS += -L $(UT_CONTROL)/lib -lut_control -Wl,-rpath-link,$(LIBWEBSOCKET_LIB_DIR):$(CURL_LIB_DIR):$(OPENSSL_LIB_DIR)
```

Comment:
remove curl & openssl, only ut-control is required here.

Reply:
A ticket was raised to  fix this in ut-control [GH#28](https://github.com/rdkcentral/ut-control/issues/28)

PR below: https://github.com/rdkcentral/ut-control/pull/29